modules/eks/aws/loki.tf

```diff
@@ -1,4 +1,5 @@
 resource "aws_s3_bucket" "loki" {
+  bucket        = "loki-${var.cluster_name}"
```

Comment:
As S3 namespace is worldwide, wouldn't this prevent having 2 cluster with the same name?
Also, maybe we should also make it overidable.

Reply:
Indeed, that's a good point. We should probably suffix the name with a random string or something like that.

Otherwise, instead of making the bucket name overridable, it might be useful to allow passing the name of an existing bucket and path, so that one could have all loki logs for different clusters (and possibly other stuff) in the same bucket. Provided that loki supports this.

Reply:
Yes, it should be overridable in any case. Maybe a good default could be `loki.${cluster_name}.${base_domain}`, this should be unique.

Reply:
We should also be able to pass a pre-created S3 bucket